roles/transmissionvpn/tasks/main.yml

```diff
@@ -0,0 +1,82 @@
+#########################################################################
+# Title:            Cloudbox Community: transmissionvpn                     #
+# Author(s):        Tarek                                               #
+# URL:              https://github.com/Cloudbox/Community               #
+# Docker Image(s):  haugene/transmission-openvpn                        #
+# --                                                                    #
+#         Part of the Cloudbox project: https://cloudbox.works          #
+#########################################################################
+#                   GNU General Public License v3.0                     #
+#########################################################################
+---
+- name: "Setting CloudFlare DNS Record"
+  include_role:
+    name: cloudflare-dns
+  vars:
+    subdomain: transmissionvpn
+  when: cloudflare_enabled
+
+- name: Stop and remove any existing container
+  docker_container:
+    name: transmissionvpn
+    state: absent
+
+- name: Create transmissionvpn directories
+  file: "path={{ item }} state=directory mode=0775 owner={{ user.name }} group={{ user.name }}"
+  with_items:
+    - /opt/transmissionvpn
+
+- name: Create new downloads directories
+  file: "path={{ item }} state=directory mode=0775 owner={{ user.name }} group={{ user.name }} recurse=yes"
+  with_items:
+    - "{{ downloads.torrents }}"
+    - "{{ downloads.torrents }}/transmissionvpn"
+    - "{{ downloads.torrents }}/transmissionvpn/completed"
+    - "{{ downloads.torrents }}/transmissionvpn/incoming"
+    - "{{ downloads.torrents }}/transmissionvpn/watched"
+    - "{{ downloads.torrents }}/transmissionvpn/torrents"
+    - /opt/transmissionvpn/
+  when: (torrents_downloads_path is defined)|default(false)
+
+- name: Set default_volumes variable
+  set_fact:
+    default_volumes:
+      - "/etc/localtime:/etc/localtime:ro"
```

Comment:
Take this out and use TZ env variable instead. Lot of issues with mounting this (ie `error: mounting folder as a file`, etc), which is why I made `{{ tz }}` a global variable to be used for docker env TZ.

Reply:
@Migz93 just one change i see that needs to be made.

Reply:
Im not sure exactly what that means! @Migz93 it will be nice if you can make it or you can explain to me and I will do it :)

Reply:
Not sure how to modify your PR directly. But i've made the changes here
https://pastebin.com/HW4yE5sc

Basically i removed this line (Line 44), from default_volumes
`- "/etc/localtime:/etc/localtime:ro"`
And instead i added this line (Line 70), to set TZ as a env option, pulling the variable from the users adv_settings file
`TZ: "{{ tz }}"`

Reply:
done